rest/route/commit_queue.go

```diff
@@ -123,8 +123,8 @@ func (cq *commitQueueDeleteItemHandler) Run(ctx context.Context) gimlet.Responde
 	}
 
 	response := gimlet.NewJSONResponse(struct{}{})
-	if err := response.SetStatus(http.StatusNoContent); err != nil {
-		return gimlet.MakeJSONInternalErrorResponder(errors.Wrapf(err, "Cannot set HTTP status code to %d", http.StatusNoContent))
+	if err := response.SetStatus(http.StatusOK); err != nil {
```

Comment:
From [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE#Responses) it looks like we'd want to return a 204.

Reply:
Oh sorry that's fine, I was just doing things with the route and was confused by the response but sounds good.